Makefile

```diff
@@ -1,19 +1,12 @@
-dev: check-env
+dev:
 	cd .. && pip3 uninstall -y sdx-common && pip3 install -I ./sdx-common
```

Comment:
Running a standard ``make; make test`` doesn't work as it assumes you already have ``sdx-common`` installed (which you won't on a fresh checkout) and make fails.

``build`` should be the default ``make`` step but that will also fail if you remove the ``git clone`` step.